src/index.ts

```diff
@@ -1,24 +1,32 @@
-// Copyright (c) 2018-2020 WalletLink.org <https://www.walletlink.org/>
 // Copyright (c) 2018-2020 Coinbase, Inc. <https://www.coinbase.com/>
 // Licensed under the Apache License, version 2.0
 
-import { WalletLinkProvider } from "./provider/WalletLinkProvider"
-import { WalletLink } from "./WalletLink"
+import { CoinbaseWalletSDK } from "./CoinbaseWalletSDK"
+import { CBWalletProvider } from "./provider/CBWalletProvider"
 
-export { WalletLinkProvider } from "./provider/WalletLinkProvider"
-export { WalletLink } from "./WalletLink"
-export default WalletLink
+export { CoinbaseWalletSDK } from "./CoinbaseWalletSDK"
+export { CBWalletProvider } from "./provider/CBWalletProvider"
+export default CoinbaseWalletSDK
 
 declare global {
   interface Window {
-    WalletLink: typeof WalletLink
-    WalletLinkProvider: typeof WalletLinkProvider
-    ethereum?: WalletLinkProvider
-    walletLinkExtension?: WalletLinkProvider
+    CoinbaseWalletSDK: typeof CoinbaseWalletSDK
+    CBWalletProvider: typeof CBWalletProvider
+    ethereum?: CBWalletProvider
+    coinbaseWalletExtension?: CBWalletProvider
+
+    // deprecated
+    WalletLink: typeof CoinbaseWalletSDK
+    WalletLinkProvider: typeof CBWalletProvider
+    walletLinkExtension?: CBWalletProvider
   }
 }
 
 if (typeof window !== "undefined") {
-  window.WalletLink = WalletLink
-  window.WalletLinkProvider = WalletLinkProvider
+  window.CoinbaseWalletSDK = CoinbaseWalletSDK
+  window.CBWalletProvider = CBWalletProvider
```

Comment:
Should be consistent with our naming? 
I've seen switches between `CBWalletSdk` and `CoinbaseWalletSdk`. 
Maybe for internal variable names we drop the `Coinbase` and `CB` and just use `WalletSdk{xx}` and for external references like here we prefix `Coinbase` to the variable name?

Reply:
+1 to this. Let's just use `WalletSdk{xx}`

Reply:
👍